docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.7'
+
+services:
+  api:
+    build: .
+    image: expdev07/coronavirus-tracker-api:2.0
+    restart: always
+    command: pipenv --python 3.7 run dev
```

Comment:
We shouldn't be running the dev version with the "reload" flag in the container.

Reply:
Done!